test/Microsoft.Health.Dicom.Web.Tests.E2E/Common/DicomInstancesManager.cs

```diff
@@ -1,4 +1,4 @@
-﻿// -------------------------------------------------------------------------------------------------
```

Comment:
Revert this change?

Reply:
git is not allowing to rollback this to previous commit.